Makefile

```
@@ -1,24 +1,30 @@
 .DEFAULT_GOAL := help
 
-.PHONY: help
+.PHONY: archive-links check clean doc doc-serve help icalendar l10n package \
+        pre-commit release-notes sbom setup snapshot test upgrade
+
+UV_RUN_CMD = uv run --no-sync
 
 help:
```

checkmake (0.3.2)

[warning] Target body for "help" exceeds allowed length of 5 lines (16).

(maxbodylength)